lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_tag_build.rb

```diff
@@ -7,7 +7,8 @@ def self.run(params)
   
         itc_ver = Fastlane::Helper::Ios::VersionHelper.get_build_version()
         int_ver = Fastlane::Helper::Ios::VersionHelper.get_internal_version() unless ENV["INTERNAL_CONFIG_FILE"].nil?
-        Fastlane::Helper::Ios::GitHelper.tag_build(itc_ver, int_ver)
+        Fastlane::Helper::GitHelper.create_tag(itc_ver)
+        Fastlane::Helper::GitHelper.create_tag(int_ver) unless int_ver.nil?
```

Comment:
Nitpick, but I wonder if it'd make sense to group the `itc_ver` and `int_ver` stuff together and only have one conditional? Might make it easier for future readers?

Reply:
Probably something I will pick up when I tackle https://github.com/wordpress-mobile/release-toolkit/issues/203